pytorch_lightning/loggers/base.py

```diff
@@ -391,10 +387,6 @@ def finalize(self, status: str) -> None:
         for logger in self._logger_iterable:
             logger.finalize(status)
 
-    def close(self) -> None:
-        for logger in self._logger_iterable:
-            logger.close()
-
```

Comment:
these can't be deleted outright. we have to first mark these as deprecated with a clear message for users, and allow time for users to migrate. https://pytorch-lightning.readthedocs.io/en/latest/governance.html#api-evolution